src/lib/utils/datetime.js

```diff
@@ -0,0 +1,7 @@
+import moment from 'moment'
+
+// Format incoming date
+export const formatDate = (date) => {
+  const momentDate = moment(date || undefined)
```

Comment:
will passing undefined fall back as `now`?

Reply:
@izgeorge it * shouldn't * ever be undefined, but yes we should have a fallback to now if its undefined